packages/vscode-extension-bpmn-editor/README.md

```diff
@@ -6,7 +6,7 @@ Create and edit BPMN and BPMN2 files.
 
 Features
 ---
-- Create and edit BPMN (`.bpmn`) and BPMN2 (`.bpmn2`) files.
+- Create and edit BPMN (`.bpmn`) and BPMN2 (`.bpmn2` or `.bpmn20.xml`) files.
```

Comment:
Create and edit BPMN (`.bpmn`) and BPMN2 (`.bpmn2` or, experimentally, `.bpmn20.xml`) files.